CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This release closes the [8.0.0 milestone](https://github.com/jessesquires/JSQMes
 - Dynamic text support. (#497, #1747) Thanks @MacMeDan!
 - Message cells now have a customizable accessory view. (#1519, #1719) Thanks @adubr!
 - Send button now can be turned on/off manually. (#1575 #1609) Thanks @sebastianludwig!
+- Video bubble now can have a custom thumbnail. (#628 #709 #1408) Thanks @weekwood @benjaminhallock
```

Comment:
One last question before making any changes. For the `JSQMessagesVideoThumbnailFactory`, this line(https://github.com/jessesquires/JSQMessagesViewController/pull/1823/files#diff-bbf85f0f48d13e18a0902a16c4af512eR71) is essential because it determines which frame to capture from the video. Should I have params to let people decide or?